notebook/tests/test_files.py

```diff
@@ -95,7 +95,7 @@ def test_contents_manager(self):
 
         r = self.request('GET', 'files/test.txt')
         self.assertEqual(r.status_code, 200)
-        self.assertEqual(r.headers['content-type'], 'text/plain; charset=UTF-8')
```

Comment:
We can define [AuthenticatedFileHandler.get_content_type](https://github.com/tornadoweb/tornado/blob/v4.5.1/tornado/web.py#L2641) to include the charset.